File: apps/auth-server/src/main/java/com/example/chat/auth/server/core/domain/MfaType.java
package com.example.chat.auth.server.core.domain;

/**
 * MFA 방식의 유형
 */
public enum MfaType {
    OTP,         // SMS, Email OTP
    TOTP,        // Time-based OTP (Google Authenticator, Authy)
    BACKUP_CODE  // 백업 코드
}

File: apps/auth-server/src/main/kotlin/com/example/chat/auth/server/core/domain/MfaType.kt

package com.example.chat.auth.server.core.domain

/**
 * MFA 방식의 유형
 * - OTP: 일회용 비밀번호 (SMS, Email, Authenticator)
 * - BACKUP_CODE: 백업 코드
 */
enum class MfaType {
    OTP,           // SMS, Email, Authenticator App
    BACKUP_CODE
}
